### ISSUE TITLE
[Request] Open apps on the workspace where they were started and not on the currently selected workspace.

### ISSUE THREAD

**Issue by Astro-City** [elementary/gala#258]
Hi.

I think that changing this behavior would benefit the Elementary work flow.

Sometimes you open an app and want to check something quick  on another workspace and you end up opening the app on the wrong workspace.

This change would avoid the above behavior and some quirks that happen when the app is opening and you're between workspace transition.

**Comment by cassidyjames** [comment]
I agree this would make sense for slower-launching apps, though I'm not sure how technically feasible it is. Can Gala know when an app is requested to open, but hasn't opened yet?
